core/src/components/range/test/a11y/range.e2e.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('range: a11y', () => {
+  test('should not have visual regressions', async ({ page, skip, browserName }) => {
+    skip.rtl();
+    skip.mode('ios', 'iOS mode does not display hover/active/focus styles.');
+
+    await page.setContent(
+      `<ion-app>
+        <ion-content>
+          <ion-range min="0" max="100" value="80"></ion-range>
+        </ion-content>
+      </ion-app>
+        `
+    );
+
+    const tabKey = browserName === 'webkit' ? 'Alt+Tab' : 'Tab';
+
+    const range = page.locator('ion-range');
+    const rangeHandle = page.locator('ion-range .range-knob-handle');
+
+    await page.keyboard.press(tabKey);
+    await page.waitForChanges();
+
+    await rangeHandle.hover();
+    await page.waitForChanges();
```

Comment:
Do we need to focus the range first to check the hover styling? I would guess we'd want to just do `rangeHandle.hover()` so we only screenshot that state.

Reply:
Good call, this was definitely the result of copy & pasting some of the test contents around 👍 hover should be checked first (without focus), then with focus, then finally when active/mouse down.